### ISSUE TITLE
Mangadex http error

### ISSUE THREAD

**Issue by RobindaCoast** [tachiyomiorg/tachiyomi-extensions#13464]
### Source information and language

Mangadex All 1.3.167

### Steps to reproduce

Try to search mangadex on the app. Get a http 503 error

### Expected behavior

Should be able to view mangadex comics and view the website in the webviewer

### Actual behavior

Unable to view or update any added mangadex comics, Mangadex in webviewer is saying the site is "under maintenance" when that isn't case on a web browser. With the Mangadex error code of:

NA-3 / ffd284e9-bd53-444e-a914-144ae2cfe669

### Tachiyomi version

0.13.6

### Android version

Android 8.1.0

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by mobi2002** [comment]
>Tachiyomi has been temporarily blocked by MangaDex due to ongoing server bugs.
